deploy/cluster_role.yaml

```diff
@@ -9,6 +9,8 @@ rules:
   resources:
   - pods
   - pods/finalizers
+  - events
+  - upgradeconfigs
```

Comment:
@chamalabey - Why are we adding these resources in cluster role? Am not sure what change requires this right now. Also, this configuration is incorrect